Simple-Store.md

```diff
@@ -0,0 +1,87 @@
+
+# Overview
+- This is a new module raincatcher-store-simple that will
+    - use current functionality in the mediator ArrayStore 
+    - use the $fh.db cloud API to store data in a mongo database
+-  The cloud applications can specify a `WFM_USE_MEMORY_STORE` environment variable 
+    - allow users to control whether the memory store or the MongoDB store is used.
+
+## Use Simple Perstistence Store for workorders in Cloud App
+
+### Create Simple Store in your Cloud App
+```js
+const SimpleStore = require('fh-wfm-simple-store');
+const simpleStore = new SimpleStore('workorders');
+simpleStore.init({/* init with data */}).then(function() {
+  // store is available for use
+});
+```
+### Registering CRUD topics for the workorders
+```js
+simpleStore.listen('cloud:', mediator);
+```
+
+### Create list of of workorders to be saved into Simple Store 
+```js
+var workorders = [
+    { id: "rkX1fdSH", workflowId: 'SyVXyMuSr', assignee: 'rkX1fdSH', type: 'Job Order', title: 'Footpath in disrepair', status: 'New', startTimestamp: '2015-10-22T14:00:00Z', address: '1795 Davie St, Vancouver, BC V6G 2M9', location: [49.287227, -123.141489], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'},
+  ...
+    { id: "BkuXajsIB", workflowId: 'SyVXyMuSr', assignee: null, type: 'Job Order', title: 'Sink in disrepair', status: 'New', startTimestamp: '2015-10-22T07:00:00Z', address: '3820 Oak St, Vancouver, BC V6H 2M5', location: [49.251362, -123.127070], summary: 'Please remove damaged kerb and SUPPLY AND FIX 1X DROP KERB CENTRE BN 125 X 150 cart away from site outside number 3.'};
+];
+```
+
+### Reset the workorder dates to today to allow viewing workorders in the scheduler 
+```js
+  var today = new Date();
+  var tomorrow = new Date();
+  tomorrow.setDate(today.getDate() + 1);
+  workorders.forEach(function(workorder, index) {
+    var date = new Date(workorder.startTimestamp);
+    var hours = date.getHours();
+    var newDate = index < workorders.length * 2 / 3 ? today : tomorrow;
+    newDate.setHours(hours);
+    workorder.startTimestamp = newDate.getTime();
+  });
+```
+
+### Publishing a create topic to create the workorders
+```js
+workorders.forEach(function(workorder) {    
+    mediator.publish('wfm:cloud:workorder:create', workorder);
+});
+```
+
+## Option to switch between array store and persistence
+In case we want to use ArrayStore then we use `WFM_USE_MEMORY_STORE` environment variable and set it to `true`
+<br><img src="assets/images/add-simple-store-env-var.png" title="Add `WFM_USE_MEMORY_STORE` Env Var" alt="Add `WFM_USE_MEMORY_STORE` Env Var" height="400px">
+
+Then push environment variables and Cloud App should be restarted
+<br><img src="assets/images/push-simple-store-env-var.png" title="Push `WFM_USE_MEMORY_STORE` Env Var" alt="Push `WFM_USE_MEMORY_STORE` Env Var" height="400px"> 
+Otherwise $fh.db store will be used for data persistence
```

Comment:
Can you move this sentence to the beginning of this section. It is hidden under the images.